.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 node_modules
+package-lock.json
```

Comment:
I'm wondering why `package-lock` is git ignored? According to the npm's doc it's intended to be committed: https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json